### PR TITLE
Add restauration plugin options with secret

### PR DESCRIPTION
**Pull request by lvenries**
Add new options to the plugin restauration

### CODE REVIEW

File: ansible/vars/secrets-wwp-test.yml [hunk 1]
```diff
@@ -50,3 +50,7 @@ wpforms:
...
   license_key: ENC[PKCS7,MIIBmQYJKoZIhvcNAQcDoIIBijCCAYYCAQAxggEhMIIBHQIBADAFMAACAQEwDQYJKoZIhvcNAQEBBQAEggEAkSUY3Ykbugwooqlkv/5DR4JK6HvCa7/5vFP3dF2C+Wz7XwLKgXLqGYiy6lmzf2FRzjVYX/x27P7LUmVuloVhNRoh2HKTwImClBaIP0JT81dWId8GrK/d91ZkI0zCx+0OuS+cfQG4JcXis+h6C9S5FkrfPFfpvMFXyPfi03kyEGJDg/EgTcYicRb5o3jYQz9LweUpbd3gBC7ro0UZ/nGggVJEuLKrbjVtJKg/XNNaftbKRymOHsjU+x6dmZlWqgrwyywrTrYi6okF2Z1dUA2LV6Gw94XdbjSnSfv4Mh1SysXWx87CMRBM6FIAQqUdTW0HwKQR1C6xuQWO6u93bwYpXzBcBgkqhkiG9w0BBwEwHQYJYIZIAWUDBAEqBBAO6pcFLHObzROzG7JWIISogDBmxA/HRj49BFPqGPF6u0kMiaGUEa8XJqzCAnYLrFkSt2XCmWPpJSISz50RYjLBPzc=]
+
+restauration:
+  api_password: ENC[PKCS7,MIIBeQYJKoZIhvcNAQcDoIIBajCCAWYCAQAxggEhMIIBHQIBADAFMAACAQEwDQYJKoZIhvcNAQEBBQAEggEAQWQxyb0HlLNnRg0knvBROHOB7OE90nM/A40Ur7Lk/iTgbHGVKGqG+fAEAL457ELMFX65qVWG8x9M0gnnJ484ercmkOA8lEmuqlINKeT0BUUjR919Xutpo4X39RYMcsJizGQ5bDrYPr9RKXigHBPaIfqgPoSuu5ZvtIloLvLgbyAtFudbaerpFqXJ1o7rN76g5W0jCec2c79Y55zroBxO4gOwU5R1zDl3Vzvfu9wV0ec5dZfBAI1LPw/Tg/vT6dZyY77+yiltCAUZyrWhpFsVFPzwq0PT/vYt+dwZM/q4eefWuwOhpRN6al16Ezx4iaPgFRYjxFtUUMBk1FQs4jCI1TA8BgkqhkiG9w0BBwEwHQYJYIZIAWUDBAEqBBBxWi9moUK9Ei003/elx8ARgBD40S0GVepcqKVJ04/wldj5]
+
```
Comment (domq) [inline ansible/vars/secrets-wwp-test.yml:56]: Please remove extra whitespace at the end

File: ansible/vars/secrets-wwp.yml [hunk 2]
```diff
@@ -35,3 +35,6 @@ wpforms:
...
   license_key: ENC[PKCS7,MIIBmQYJKoZIhvcNAQcDoIIBijCCAYYCAQAxggEhMIIBHQIBADAFMAACAQEwDQYJKoZIhvcNAQEBBQAEggEAUOxNjo8UrcJQNeDCt46KTgoFJfPttzTLA+WVpzmz6RcgKr93VTDHsLFHhFBVsFbo/wNhsYJo5t6N37A+B3S07SMIPIi5+G3BuBqtqAi5pyOsA0Y0Vnzkov6rSCuuXk2U7+9nxhswQ5rRhbF+lE2eBhsJB4rPF8QWW2l4dyaLckCGlXGd2lq6oRSuqTqr19HVJQ30CwwnZEqoW2ps5qXajxnrlLVIZkQq3o5yfSRZghB48J3lJI6UOGpS8hGoxnJTS/D0rApj4sDfpV4Dyjyat134xm47oLADdm7sJC+HliBC3vzmPFqQ7LMDssVCHM6mWP6jNDiWC7OpskhoTiZRcjBcBgkqhkiG9w0BBwEwHQYJYIZIAWUDBAEqBBAJL4kt5Et7A1rxu0XOKgQ2gDBJeik2AOPFpF7LuBKQhkvrZBg8zxMiyv3oMe7wQ1KX3PM94zsGc6u4P3i7NJ9CQYY=]
+
+restauration:
+  api_password: ENC[PKCS7,MIIBeQYJKoZIhvcNAQcDoIIBajCCAWYCAQAxggEhMIIBHQIBADAFMAACAQEwDQYJKoZIhvcNAQEBBQAEggEAQWQxyb0HlLNnRg0knvBROHOB7OE90nM/A40Ur7Lk/iTgbHGVKGqG+fAEAL457ELMFX65qVWG8x9M0gnnJ484ercmkOA8lEmuqlINKeT0BUUjR919Xutpo4X39RYMcsJizGQ5bDrYPr9RKXigHBPaIfqgPoSuu5ZvtIloLvLgbyAtFudbaerpFqXJ1o7rN76g5W0jCec2c79Y55zroBxO4gOwU5R1zDl3Vzvfu9wV0ec5dZfBAI1LPw/Tg/vT6dZyY77+yiltCAUZyrWhpFsVFPzwq0PT/vYt+dwZM/q4eefWuwOhpRN6al16Ezx4iaPgFRYjxFtUUMBk1FQs4jCI1TA8BgkqhkiG9w0BBwEwHQYJYIZIAWUDBAEqBBBxWi9moUK9Ei003/elx8ARgBD40S0GVepcqKVJ04/wldj5]
```
Comment (domq) [inline ansible/vars/secrets-wwp.yml:40]: Please be sure to end all files with a newline.

File: ansible/vars/secrets-wwp-test.yml [hunk 3]
```diff
@@ -50,3 +50,7 @@ wpforms:
...
   license_key: ENC[PKCS7,MIIBmQYJKoZIhvcNAQcDoIIBijCCAYYCAQAxggEhMIIBHQIBADAFMAACAQEwDQYJKoZIhvcNAQEBBQAEggEAkSUY3Ykbugwooqlkv/5DR4JK6HvCa7/5vFP3dF2C+Wz7XwLKgXLqGYiy6lmzf2FRzjVYX/x27P7LUmVuloVhNRoh2HKTwImClBaIP0JT81dWId8GrK/d91ZkI0zCx+0OuS+cfQG4JcXis+h6C9S5FkrfPFfpvMFXyPfi03kyEGJDg/EgTcYicRb5o3jYQz9LweUpbd3gBC7ro0UZ/nGggVJEuLKrbjVtJKg/XNNaftbKRymOHsjU+x6dmZlWqgrwyywrTrYi6okF2Z1dUA2LV6Gw94XdbjSnSfv4Mh1SysXWx87CMRBM6FIAQqUdTW0HwKQR1C6xuQWO6u93bwYpXzBcBgkqhkiG9w0BBwEwHQYJYIZIAWUDBAEqBBAO6pcFLHObzROzG7JWIISogDBmxA/HRj49BFPqGPF6u0kMiaGUEa8XJqzCAnYLrFkSt2XCmWPpJSISz50RYjLBPzc=]
+
+restauration:
+  api_password: ENC[PKCS7,MIIBeQYJKoZIhvcNAQcDoIIBajCCAWYCAQAxggEhMIIBHQIBADAFMAACAQEwDQYJKoZIhvcNAQEBBQAEggEAQWQxyb0HlLNnRg0knvBROHOB7OE90nM/A40Ur7Lk/iTgbHGVKGqG+fAEAL457ELMFX65qVWG8x9M0gnnJ484ercmkOA8lEmuqlINKeT0BUUjR919Xutpo4X39RYMcsJizGQ5bDrYPr9RKXigHBPaIfqgPoSuu5ZvtIloLvLgbyAtFudbaerpFqXJ1o7rN76g5W0jCec2c79Y55zroBxO4gOwU5R1zDl3Vzvfu9wV0ec5dZfBAI1LPw/Tg/vT6dZyY77+yiltCAUZyrWhpFsVFPzwq0PT/vYt+dwZM/q4eefWuwOhpRN6al16Ezx4iaPgFRYjxFtUUMBk1FQs4jCI1TA8BgkqhkiG9w0BBwEwHQYJYIZIAWUDBAEqBBBxWi9moUK9Ei003/elx8ARgBD40S0GVepcqKVJ04/wldj5]
```
Comment (domq) [inline ansible/vars/secrets-wwp-test.yml:55]: As discussed, you will need to re-encrypt the same secret with the key that is specific to this file.

**Review by domq — CHANGES_REQUESTED** [review]

File: ansible/vars/secrets-wwp-test.yml (hunk 3 above)
Comment (lvenries) [reply ansible/vars/secrets-wwp-test.yml]: the secret is re-encrypt with test key

**Review by domq — APPROVED** [review]

**Comment by domq** [comment]
👍

**Pull request merged**